stories/tabs.stories.js

```diff
@@ -58,3 +58,60 @@ stories.add(
   ),
   { notes: { markdown: notes } }
 );
+
+stories.add(
+  'Controlled',
```

Comment:
This basically behaves just like the Default (uncontrolled) story, but the source demonstrates using Tabs in controlled mode.